.github/workflows/firestore.yml

```diff
@@ -20,6 +20,9 @@ on:
     # Firestore sources
     - 'Firestore/**'
 
+    # Interop headers
+    - 'Interop/Auth/Public/*.h'
```

Comment:
I'm not sure. I've been avoiding adding all the transitive deps to avoid CI backlogs in favor of forcing build all's via a Gemfile change. The last time I researched I saw that GHA was planning to add a feature to force all tests to run, but it was not there yet.